src/collection.cpp

```diff
@@ -1963,6 +1963,7 @@ Option<bool> Collection::init_index_search_args(collection_search_args_t& coll_a
     const text_match_type_t& match_type = coll_args.match_type;
     const size_t& facet_sample_percent = coll_args.facet_sample_percent;
     const size_t& facet_sample_threshold = coll_args.facet_sample_threshold;
+    const size_t& facet_sample_slope = coll_args.facet_sample_slope;
```

Comment:
You also have to add it here, or it won't work in the API:

https://github.com/typesense/typesense/blob/598e17838c39bd585474d8bba3a72472a212c7f0/src/collection.cpp#L8426